python/map_closures/pipeline.py

```diff
@@ -109,7 +109,10 @@ def run(self):
         self._log_to_console()
         self._write_data_to_disk()
 
-        return self.results
+        if self._eval:
```

Comment:
With the change above, this is no longer necessary, just returning `self.results` is sufficient again.